packages/p2p-v2/src/pages/orders/components/ChatMessages/ChatMessages.tsx

```diff
@@ -29,44 +29,74 @@ const AdminMessage = () => (
+            // Scroll all the way to the bottom of the container.
+            scrollRef.current.scrollTop = scrollRef.current.scrollHeight;
+        }
+    }, [chatMessages.length]);
+
     const getMessageFormat = (chatMessage: TChatMessages[number], messageColor: string) => {
         const { fileType = '', name, size = 0, url } = chatMessage ?? {};
-        if (isImageType(fileType))
-            return (
-                <a className='p2p-v2-chat-messages__item__image' href={url} rel='noopener noreferrer' target='_blank'>
-                    <img alt={name} src={url} />
-                </a>
-            );
-        else if (isPDFType(fileType)) {
-            return (
-                <ChatMessageText color={messageColor}>
-                    <div className='p2p-v2-chat-messages__item__pdf'>
-                        <PDFIcon />
-                        <a href={url} rel='noopener noreferrer' target='_blank'>
+        switch (fileType) {
+            case CHAT_FILE_TYPE.IMAGE:
+                return (
+                    <a
+                        className='p2p-v2-chat-messages__item__image'
+                        href={url}
+                        rel='noopener noreferrer'
+                        target='_blank'
+                    >
+                        <img alt={name} onLoad={onImageLoad} src={url} />
+                    </a>
+                );
+            case CHAT_FILE_TYPE.PDF:
+                return (
+                    <ChatMessageText color={messageColor}>
+                        <div className='p2p-v2-chat-messages__item__pdf'>
+                            <PDFIcon />
+                            <a href={url} rel='noopener noreferrer' target='_blank'>
+                                {name}
+                            </a>
+                        </div>
+                        {`${convertToMB(size).toFixed(2)}MB`}
+                    </ChatMessageText>
+                );
+
+            default:
+                return (
+                    <ChatMessageText color={messageColor}>
+                        <a
+                            className='p2p-v2-chat-messages__item__file'
+                            href={url}
+                            rel='noopener noreferrer'
+                            target='_blank'
+                        >
                             {name}
                         </a>
-                    </div>
-                    {`${convertToMB(size).toFixed(2)}MB`}
-                </ChatMessageText>
-            );
+                    </ChatMessageText>
+                );
+        }
+    };
+
+    const onImageLoad = (event: SyntheticEvent<HTMLImageElement, Event>) => {
+        // Height of element changes after the image is loaded. Accommodate
+        // this extra height in the scroll.
+        if (scrollRef.current) {
+            scrollRef.current.scrollTop += (event.currentTarget.parentNode as HTMLElement)?.clientHeight;
```

Comment:
Does this work here @nada-deriv, because if the element is `null` or `undefined`, then we should add nothing to the scrollTop? 🤔 
Also, based on [this](https://stackoverflow.com/questions/8685739/difference-between-dom-parentnode-and-parentelement), it seems safe to use `parentElement`, unless the user is using `Firefox 9` or below, which is quite unlikely 😅.
```suggestion
            scrollRef.current.scrollTop += event.currentTarget.parentElement
                ? event.currentTarget.parentElement.clientHeight
                : 0;
```